pyk4a/config.py

```diff
@@ -85,13 +88,27 @@ def __init__(self,
         self.subordinate_delay_off_master_usec = subordinate_delay_off_master_usec
         self.disable_streaming_indicator = disable_streaming_indicator
 
-    def unpack(self):
-        return (self.color_format,
-                self.color_resolution,
-                self.depth_mode,
-                self.camera_fps,
-                self.synchronized_images_only,
-                self.depth_delay_off_color_usec,
-                self.wired_sync_mode,
-                self.subordinate_delay_off_master_usec,
-                self.disable_streaming_indicator)
+    def unpack(
```

Comment:
Is there anyway to make this less ugly without cheating the formatting tools?

Reply:
I made line-lenght 120 chars. Now it looks better